pstree.go

```diff
@@ -71,60 +99,32 @@ func scan(dir string) (Process, error) {
 	}
 	defer f.Close()
 
-	// see: http://man7.org/linux/man-pages/man5/proc.5.html
-	stat := struct {
-		pid       int    // process ID
-		comm      string // filename of the executable in parentheses
-		state     byte   // process state
-		ppid      int    // pid of the parent process
-		pgrp      int    // process group ID of the process
-		session   int    // session ID of the process
-		tty       int    // controlling terminal of the process
-		tpgid     int    // ID of foreground process group
-		flags     uint32 // kernel flags word of the process
-		minflt    uint64 // number of minor faults the process has made which have not required loading a memory page from disk
-		cminflt   uint64 // number of minor faults the process's waited-for children have made
-		majflt    uint64 // number of major faults the process has made which have required loading a memory page from disk
-		cmajflt   uint64 // number of major faults the process's waited-for children have made
-		utime     uint64 // user time in clock ticks
-		stime     uint64 // system time in clock ticks
-		cutime    int64  // children user time in clock ticks
-		cstime    int64  // children system time in clock ticks
-		priority  int64  // priority
-		nice      int64  // the nice value
-		nthreads  int64  // number of threads in this process
-		itrealval int64  // time in jiffies before next SIGALRM is sent to the process dure to an interval timer
-		starttime int64  // time the process started after system boot in clock ticks
-		vsize     uint64 // virtual memory size in bytes
-		rss       int64  // resident set size: number of pages the process has in real memory
-	}{}
-
+	var stat ProcessStat
```

Comment:
```suggestion
	var proc Process
```
and then fill directly `proc.Stat` below.